conda/exception_handler.py

```diff
@@ -174,6 +174,15 @@ def print_unexpected_error_report(self, error_report):
                 "An unexpected error has occurred. Conda has prepared the above report."
             )
             message_builder.append("")
+            message_builder.append(
+                "If you suspect this error is being caused by a malfunctioning plugin,"
+            )
+            message_builder.append(
+                "consider using the --no-plugins option to turn off external plugins."
```

Comment:
I wonder if users understand what "external" means, can we somehow use "built-in"?

```suggestion
                "consider using the --no-plugins option to only enable built-in plugins."
```